pkg/util/mount/mount.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/exec"
 )
 
-type Interface interface {
+type Mounter interface {
```

Comment:
We should probably have kept `volume.Host` and `volume.Plugin` without the
redundant Volume in there.  I would approve such a PR

On Thu, Sep 10, 2015 at 9:46 PM, Paul Morie <notifications@github.com>
wrote:

> In pkg/util/mount/mount.go
> <https://github.com/kubernetes/kubernetes/pull/13791#discussion_r39240979>
> :
>
> > @@ -25,7 +25,7 @@ import (
> >  	"k8s.io/kubernetes/pkg/util/exec"
> >  )
> >
> > -type Interface interface {
> > +type Mounter interface {
>
> @thockin <https://github.com/thockin> Yeah, agree that we probably don't
> need this name change. I actually have suggested this name change myself
> before. It is idiomatic go to use Interface for the primary interface a
> package provides and descriptive names for secondary interfaces.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13791/files#r39240979>.
>